plugins/diff/erofs/differ_linux.go

```diff
@@ -245,6 +268,20 @@ func (s erofsDiff) Apply(ctx context.Context, desc ocispec.Descriptor, mounts []
 	}
 	defer ra.Close()
 
+	layerBlobPath := path.Join(layer, "layer.erofs")
+	if native {
+		// TODO: prefer to use hardlink as an optimization here
+		f, err := os.Create(layerBlobPath)
+		if err != nil {
+			return emptyDesc, err
+		}
+		_, err = io.Copy(f, content.NewReader(ra))
```

Comment:
`f` should be closed unconditionally after this call

Reply:
fixed.